### PR TITLE
fix: [CI-9657]: logs download fix

### PR DESCRIPTION
**Pull request by devkimittal**
fix: [CI-9657]: Add support for jwt from manager secret

token fix

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

[CI-9657]: https://harness.atlassian.net/browse/CI-9657?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/56394)

### CODE REVIEW

File: product/platform/client/http.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	apiKeyEndpoint    = "/token/validate?accountIdentifier=%s"
-	accountEndpoint   = "/gateway/api/account/%s"
+	accountEndpoint   = "accounts/%s"
```
Comment (smjt-h) [inline product/platform/client/http.go:30]: keep it as /accounts

File: product/log-service/config/config.go [hunk 2]
```diff
@@ -29,6 +29,7 @@ type Config struct {
 		ACLBaseURL       string        `envconfig:"LOG_SERVICE_PLATFORM_ACCESS_CONTROL_BASE_URL" default:"http://localhost:9006/api"`
 		VanityURLEnabled bool          `envconfig:"LOG_SERVICE_VANITY_URL_ENABLED"`
 		VanityURLTTL     time.Duration `envconfig:"LOG_SERVICE_VANITY_URL_TTL" default:"1h"`
+		ManagerSecret    string        `envconfig:"LOG_SERVICE_MANAGER_SECRET" default:"dOkdsVqdRPPRJG31XU0qY4MPqmBBMk0PTAGIKM6O7TGqhjyxScIdJe80mwh5Yb5zF3KxYBHw6B3Lfzlq"`
```
Comment (smjt-h) [inline product/log-service/config/config.go:32]: Its an ng-manager secret?

Comment (smjt-h) [reply product/log-service/config/config.go:32]: jwtAuthSecret

**Comment by devkimittal** [comment]
related config change https://github.com/wings-software/ng-prod-manifests/pull/5268

**Review by smjt-h — APPROVED** [review]

**Pull request merged**